### PR TITLE
PMM-9994 Change QAN description.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Review by dygksy — APPROVED** [review]

File: docs/using/query-analytics.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The dashboard contains three panels:
 
 ![!image](../_images/PMM_Query_Analytics_Panels_Filters.jpg)
 
-- The Filter panel occupies the left side of the dashboard. It lists filters, grouped by category. Selecting one reduces the Overview list to those items matching the filter.
+- The Filter panel occupies the left side of the dashboard. It lists filters, grouped by category and also shows the percentage of main metric that is explained below. If you select different metric as main then percentages on left panel will be different and related to this metric. Selecting one reduces the Overview list to those items matching the filter.
```
Comment (rasika-chivate) [inline docs/using/query-analytics.md:27]:
```suggestion
- The Filter panel on the left side of the dashboard lists filters grouped by category. It also shows the percentage of the main metric, which is explained below. If you select a different main metric, the percentages on the left panel will be related to this metric. Selecting one reduces the Overview list to those items matching the filter.
```

Comment (rasika-chivate) [reply docs/using/query-analytics.md:27]: @JiriCtvrtka, I did not understand this sentence "Selecting one reduces the Overview list to those items matching the filter."

**Review by rasika-chivate — CHANGES_REQUESTED** [review]

File: docs/using/query-analytics.md (hunk 1 above)
Comment (JiriCtvrtka) [reply docs/using/query-analytics.md:27]: @rasika-chivate Like when you pick filter item then overview and count is changed to matching filter. So in results you can see different count of rows and different count of items. Could you help me redefine it to make it clear?

File: docs/using/query-analytics.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The dashboard contains three panels:
 
 ![!image](../_images/PMM_Query_Analytics_Panels_Filters.jpg)
 
-- The Filter panel occupies the left side of the dashboard. It lists filters, grouped by category. Selecting one reduces the Overview list to those items matching the filter.
+- The Filter panel on the left side of the dashboard lists filters grouped by category. It also shows the percentage of the main metric, which is explained below. If you select a different main metric, the percentages on the left panel will be related to this metric. Selecting one reduces the Overview list to those items matching the filter.
```
Comment (rasika-chivate) [inline docs/using/query-analytics.md:27]:
```suggestion
- The Filter panel on the left side of the dashboard lists the filters grouped by category. It also shows the percentage of the main metrics (explained below). If you select a different metric, the percentages on the left panel will change as per this metric. When you select a metric, it reduces the overview list as per the matching filter.





[4:33](https://perconacorp.slack.com/archives/D02VBHC55U0/p1661166190044839)
Please let me know. Then I will merge the PR...
```

**Review by rasika-chivate — CHANGES_REQUESTED** [review]

**Review by rasika-chivate — APPROVED** [review]

**Pull request merged**